aggregation/simple/src/main/java/com/spotify/heroic/aggregation/simple/MetricMappingAggregation.java

```diff
@@ -41,10 +42,15 @@
 import java.util.Map;
 import java.util.Set;
 import lombok.Data;
+import lombok.NonNull;
 
 @Data
 public abstract class MetricMappingAggregation implements AggregationInstance {
+
     private static final EmptyInstance INNER = EmptyInstance.INSTANCE;
+
+    @NonNull
```

Comment:
`NonNull` is not needed?

Reply:
Copied from here: https://github.com/OdenTech/heroic/blob/fix-filter-points-agg/aggregation/simple/src/main/java/com/spotify/heroic/aggregation/simple/FilterAggregation.java#L52

Reply:
Yeah OK. It's mostly harmless so I'm OK with it. Here it would generate a null-check in the constructor which we (should) already disallow in serialization.